lib/fake_ifdh.py

```diff
@@ -234,6 +234,10 @@ def generate_proxy_command_verbose_args(cmd_str: str) -> Dict[str, Any]:
 
 
 def fix_pnfs(path: str) -> str:
+
```

Comment:
Would it be possible to have a unit test or two added to make sure this works as we expect (I mean, it SHOULD, but...)

Reply:
On it...

Reply:
Added.

Reply:
Thanks!